test/TestStableToken.t.sol

```diff
@@ -164,13 +163,45 @@ contract TestStableTokenTest is Test {
         assertTrue(token.isMinter(user2));
     }
 
-    function test__ERC20BasicFunctionality() external {
+    function test__MinterAddedEventEmitted() external {
+        vm.expectEmit(true, true, false, false);
+        emit MinterAdded(user1);
+
         vm.prank(owner);
         token.addMinter(user1);
+    }
+
+    function test__MinterRemovedEventEmitted() external {
+        vm.prank(owner);
+        token.addMinter(user1);
+
+        vm.expectEmit(true, true, false, false);
+        emit MinterRemoved(user1);
+
+        vm.prank(owner);
+        token.removeMinter(user1);
+    }
+
+    event MinterAdded(address indexed account);
+    event MinterRemoved(address indexed account);
+    event ETHBurned(uint256 amount, address indexed minter, address indexed to, uint256 tokensMinted);
```

Comment:
Nit: I'd move these up. Unexpected to see event definitions somewhere between tests.